data/lisbon.yaml

```diff
@@ -0,0 +1,35 @@
+---
+info:
+  city: Lisbon
+  link: lisbon
+  date: 12/17/2019
+  hour: '19:00'
+  icon: icons/lisbon/lisbon.svg
+  iconHover: icons/lisbon/lisbon-hover.svg
+  hostIcon: icons/paris/dashlane.svg
+  hostName: Dashlane
+  maxCapacity: 25
+site:
+  city: Paris
```

Comment:
I think this needs to be updated to Lisbon 😊

Reply:
Changed :)